doc/motr-in-prose.md

```diff
@@ -125,8 +125,9 @@ network messages.
 Distributed transaction manager ([dt](#dtm-idea)) allows operations to be 
 grouped into distributed transactions, which are guaranteed to be atomic in 
 the face of certain failures.
-
-<img src="/doc/Images/1-Distributed-transaction-manager.JPG" width=600 height=500>
+<p align="center">
+<img src="/doc/Images/1-Distributed-transaction-manager.JPG" width=400 height=300>
+</p>
```

Comment:
This does not look right. Markdown, by design, does not specify the placement of textual and graphical elements: these decisions are left to the various markdown "processors" (like html backend). Markdown documents are supposed to be readable as text, and adding html markup to them makes them unreadable.

Reply:
@andriytk, tagging you to have a thought on Nikita's suggestion.

Reply:
@nikitadanilov, do you have any suggestion on how to make them look better in github processor? (Which is the one most people use by default most of the time to read the document.)